themes/src/main/resources/theme/base/admin/resources/partials/realm-tokens.html

```diff
@@ -152,4 +152,4 @@
 
 </div>
 
-<kc-menu></kc-menu>
```

Comment:
Hi, just came back from 2 days out of the computer. Ok, I'll try to revert this one now... Cheers